README.md

```diff
@@ -671,34 +671,42 @@ around.
 
 ## Remote Terraform configurations
 
-Terragrunt has the ability to work with Terraform configurations that are not in your working directory. This is 
-especially useful if you have versioned all of your Terraform code and want to `apply` a specific version of it
-(see [Using Pipelines to Manage Environments with Infrastructure as 
-Code](https://medium.com/@kief/https-medium-com-kief-using-pipelines-to-manage-environments-with-infrastructure-as-code-b37285a1cbf5)).
-
-In your `.terragrunt` file, you can specify a `terraform` block with a `source` parameter that tells Terragrunt where 
-to find the Terraform configuration files:
+A common problem with Terraform is figuring out how to minimize copy/paste between environments (i.e. stage, prod). For
+example, consider the following file structure:
 
-```hcl
-terraform {
-  source = "git::git@github.com:foo/bar.git//my-terraform-configurations?ref=v0.0.1"
-}
+```
+infrastructure-live
+  └ stage
+    └ frontend-app
+        └ main.tf
+        └ vars.tf
+        └ outputs.tf
+    └ backend-app
+    └ search-app
+    └ mysql
+    └ redis
+    └ vpc
+  └ prod
+    └ frontend-app
+        └ main.tf
+        └ vars.tf
+        └ outputs.tf
+    └ backend-app
+    └ search-app
+    └ mysql
+    └ redis
+    └ vpc
 ```
 
-When Terragrunt finds a `terraform` block, it will:
- 
-1. Download the configurations specified via the `source` parameter into a temporary folder. This is done using the
-   [terraform init command](https://www.terraform.io/docs/commands/init.html), so the `source` parameter supports the
-   exact same syntax as the [module source](https://www.terraform.io/docs/modules/sources.html) parameter, including
-   local file paths, Git URLs, and Git URLs with `ref` parameters (useful for checking out a specific tag, commit, or
-   branch of Git repo).
-1. Copy all files from the current working directory into the temporary folder. This is useful if you have `.tfvars`
-   files in your local path that you are using to set variables within your templates.
-1. Execute whatever Terraform command you specified in that temporary folder. **Note**: if you are passing any file
-   paths (other than paths to files in the current working directory) to Terraform via command-line options, those 
-   paths must be absolute paths since we will be running Terraform from the temporary folder!
+For each environment, you have to copy/paste `main.tf`, `vars.tf`, and `outputs.tf` for each component (e.g. 
+frontend-app, backend-app, vpc, etc). As the number of components and environments grows, having to maintain more and 
+more code can become error prone. You can significantly reduce the amount of copy paste using [Terraform 
+modules](https://blog.gruntwork.io/how-to-create-reusable-infrastructure-with-terraform-modules-25526d65f73d), but even 
+the code to use the module module and set up input variables, output variables, providers, and remote state can still
```

Comment:
Fixed that in a [follow-up commit](https://github.com/gruntwork-io/terragrunt/pull/111/commits/796fae2f95265ffd0dca77d721baa067bcce41dd) :)